src/microbe_stage/Microbe.cs

```diff
@@ -997,31 +1006,14 @@ public void Kill()
         // Some pre-death actions are going to be run now
     }
 
-    public void PlaySoundEffect(string effect)
+    public void PlaySoundEffectOnce(string effect)
```

Comment:
I think this is unnecessary. Our sound file configuration can take care of things being non-looping. Otherwise this type of code is needed for *every* game entity type (and also later stages). So I don't like this change. Also, we are modifying the returned Godot loaded object, which *might* be the same instance if Godot is caching things properly, so the "loop" parameter is fundamentally broken. Instead the change needs to be in the Godot import settings per sound file.

Reply:
I dont understand why this change was made? If it was to underline the fact that the sound is played once im good with it.

Reply:
The hope here was to not have the sound file be responsible for knowing _how_ the sound is played. Currently our .ogg files are hard-coded to either loop or not loop, and the play sound effect method has no control over it. I feel that that's very unintuitive and causes issues if we want the same sound to sometimes loop and sometimes not, but hhyyrylainen makes a very good point with the caching, so I'm still trying to think of a better solution.

Reply:
Bring it up with the Godot developers. That's how the engine is designed. I was confused as well, but I got over it. This is just a bad design decision in the engine and we just have to go with it (see my comments above).